src/PseudoTypes/List_.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types = 1);
+
+namespace phpDocumentor\Reflection\PseudoTypes;
+
+use phpDocumentor\Reflection\Type;
+use phpDocumentor\Reflection\Types\Array_;
+use phpDocumentor\Reflection\Types\Integer;
+use phpDocumentor\Reflection\Types\Mixed_;
+
+final class List_ extends Array_
```

Comment:
Could you please implement the `\phpDocumentor\Reflection\PseudoType` interface for this type? This will make the list type aligned with what we have for the other pseudotypes.